persistence-modules/spring-boot-h2/spring-boot-h2-database/src/main/java/com/baeldung/naming/HibernateConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.naming;
+
+import org.hibernate.jpa.boot.spi.IntegratorProvider;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class HibernateConfig implements HibernatePropertiesCustomizer {
```

Comment:
Thanks for your explanation in the other PR that this is for convenience in operating the demonstration. Let's place a comment block at the class level here explaining that. The reason is so that folks don't inadvertently use it, thinking it is necessary.

Reply:
Ok, I'll add the comment.